app/controllers/pacs_controller.rb

```diff
@@ -0,0 +1,24 @@
+class PacsController < EmbeddedToolsController
+  before_action :authenticate, if: :authentication_required?
+
+  def exclude_syndicated_iframe_resizer?
+    true
+  end
+
+  protected
+
+  def authenticate
+    authenticate_or_request_with_http_basic do |username, password|
+      Authenticable.authenticate(username, password)
+    end
+  end
+
+  def authentication_required?
+    Authenticable.staging? || Rails.env.production?
+  end
+
+  def category_id
+    nil
+  end
+  helper_method :category_id
+end
```

Comment:
I know it's meant to be temporary the authentication bit, does it mean we are ok to deploy it without a test to validate it? 😞

Reply:
I get that @tomas-stefano, I personally think that testing is just part of our workflow, no matter if the code is temporary (unfortunately), even if I've raised the question mainly because we don't want to expose this by accident before launch. But if as you said you tested manually and feel comfortable with it, that's fair enough to me. 👍

Reply:
Yes I agree with you, tests should be part of our workflow. I don't want to make a rule though about temporary things to not have tests because it will depend on each case. I think for the nature of this "Have a temporary http authentication until the data is confirmed by the banks that is accurate" it sounds very heavy to me that I shouldn't add tests for this. 
Other case to mention, If the business came with "temporary" things then I would not trust as much of the temporary and I would probably add tests, but PACS needs to be public at some point which makes me feel that's the reasonable approach.

Reply:
I'm not sure I agree with not having tests for this feature and the reasons given do not convince me otherwise; plus it's a dangerous precedent to set. I would have preferred to have a discussion or a bit more time to think it through. I also think that when going against the agreed code practices of the team, it would be better to discuss with the whole team. For one thing, it would be a very good way of stating that this is not setting a precedent.

Reply:
Probably I shouldn't have approved it with an ongoing discussion :bowing_man:

Reply:
I understand everyone, my intentions was not to set a precedent as well. I put this on the commit messages believing that everybody will read but I think I failed on that assumption. I will move to PR description before.
But I really like to heard what are the flaws of my arguments (but be nice please!). Anyway, if everyone still wants a test I am happy to add a PR adding one, even if I disagree.

Reply:
By the way, sorry If I merged this PR. The time that I merged the PR that was two comments only and Giuseppe approved so then I merged the PR, but I am happy to talk about this.